src/multiplots.ts

```diff
@@ -1277,15 +1278,21 @@ export class MultiplePlots {
         })
 
         if (subplot instanceof ParallelPlot) {
-          subplot.rubber_bands.forEach((rubberBand) => {
-            const currentRubberIndex = rubberBandNames.indexOf(rubberBand.attributeName)
-            if (currentRubberIndex != -1) {
-              rubberBand.axisMin = rubberBands[currentRubberIndex].axisMin;
-              rubberBand.axisMax = rubberBands[currentRubberIndex].axisMax;
-              rubberBand.minValue = rubberBands[currentRubberIndex].minValue;
-              rubberBand.maxValue = rubberBands[currentRubberIndex].maxValue;
-            }
-          })
+          if (subplot !== currentPP) {
```

Comment:
Yes it is. For the moment, I want to modify only mechanisms from specific plots.
I began with histogram cause it is a frame plot with rubberbands, then i'll do scatter, then i'll do pp. I'll do this at this time, when it will not be required anymore since all these methods will be in BasePlot